### ISSUE TITLE
2021-9-2

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#600]
321There are 6 files in folder, including readme; (2021-8-31)
322What is python PIL? Python Imaging Library; Python Imaging Library is a free and open-source additional library for the Python programming language that adds support for opening, manipulating, and saving many different image file formats. It is available for Windows, Mac OS X and Linux. The latest version of PIL is 1.1.7, was released in September 2009 and supports Python 1.5.2–2.7; (2021-8-31)
323Out of memory; set batch size 32 is okay; (2021-8-31)
324What is ResNet 残差网络? 1.1 梯度弥散/爆炸; 1.2 网络退化问题; 残差单元可以以跳层连接的形式实现; https://zhuanlan.zhihu.com/p/80226180 (2021-8-31)
325Result is correct; (2021-9-1)
326In computer science, incremental learning is a method of machine learning in which input data is continuously used to extend the existing model's knowledge i.e. to further train the model. It represents a dynamic technique of supervised learning and unsupervised learning that can be applied when training data becomes available gradually over time or its size is out of system memory limits. Algorithms that can facilitate incremental learning are known as incremental machine learning algorithms. (2021-9-1)
327Model contain: beforeTrain(); train(); afterTrain(); pkl files are saved model; 连续学习就是incremental learning; (2021-9-1)
328What is nn.linear? CLASStorch.nn.Linear(in_features, out_features, bias=True, device=None, dtype=None) ; Applies a linear transformation to the incoming data; (2021-9-1)
329model.eval() is a kind of switch for some specific layers/parts of the model that behave differently during training and inference (evaluating) time. For example, Dropouts Layers, BatchNorm Layers etc. You need to turn off them during model evaluation, and .eval() will do it for you. In addition, the common practice for evaluating/validation is using torch.no_grad() in pair with model.eval() to turn off gradients computation: (2021-9-1)
